animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -276,16 +286,19 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
         iSin = sin(startAngle)
         stepCos = cos(angleStep)
         stepSin = sin(angleStep)
+        
+        Zoffset = altitude / 2 * int(self.centerAxis)
 
         for i in range(amount):
             f = <float>i * factor
 
             size = f * (endSize - startSize) + startSize
-            radius = f * (endRadius - startRadius) + startRadius
+            radius = (endRadius - startRadius) * interpolation_1(max(min(f, 1.0), 0.0)) + startRadius
+            Zstep = altitude * interpolation_2(max(min(f, 1.0), 0.0))
```

Comment:
Min and max are not needed here as well. You can just inline this variable as well, no need to store this value in a variable.

Interpolations in Cython should be evaluated using the `evaluate` method instead.